integration-tests/common/metric.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import "encoding/json"
+
+type MetricResponse struct {
+	Status string     `json:"status"`
+	Data   MetricData `json:"data"`
+}
+
+type MetricData struct {
+	ResultType string         `json:"resultType"`
+	Result     []MetricResult `json:"result"`
+}
+
+type MetricResult struct {
+	Metric Metric `json:"metric"`
+}
+
+type Metric struct {
+	Job  string `json:"job"`
+	Name string `json:"__name__"`
```

Comment:
Similar, seems like a good idea to also check the metric value just for completeness. We can assert with something like `GreaterThan`.